src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/ImpactVector.java

```diff
@@ -0,0 +1,51 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.decisionmaker.actions;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ImpactVector {
+
+    public enum Resource {
```

Comment:
should we rename this enum to something like "ResourceVector" ? we already have a Resource class defined in protobuf

Reply:
Renamed to `Dimension`